source/appModuleHandler.py

```diff
@@ -39,7 +39,7 @@
 from fileUtils import getFileVersionInfo
 
 # Dictionary of processID:appModule pairs used to hold the currently running modules
-runningTable: Dict[int, AppModule] = {}
+runningTable: Dict[int, "AppModule"] = {}

```

Comment:
Why this change is here? Note that it is unnecessary on master since type hints in `appModuleHandler` are evaluated lazily.

Reply:
These were necessary on `rc`, and had to be added. I've removed this from the merge.